rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java

```diff
@@ -45,37 +45,45 @@ public Optional<Coin> getLockingCap() {
     }
 
     @Override
-    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) {
+    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) throws LockingCapIllegalArgumentException {
+        final String INCREASE_LOCKING_CAP_TAG = "[increaseLockingCap] %s";
+        String baseMessage = String.format(INCREASE_LOCKING_CAP_TAG, " {} {}");
+        if (newLockingCap.getValue() <= 0) {
+            String message = "Locking Cap must be greater than zero. Value attempted: ";
+            logger.warn(baseMessage, message, newLockingCap.value);
+            throw new LockingCapIllegalArgumentException(message + newLockingCap.value);
+        }
+
         // Only pre-configured addresses can modify Locking Cap
         AddressBasedAuthorizer authorizer = constants.getIncreaseAuthorizer();
         if (!authorizer.isAuthorized(tx, signatureCache)) {
-            logger.warn("[increaseLockingCap] {} {}", "Not authorized address tried to increase Locking Cap. Address: ", tx.getSender(signatureCache));
+            logger.warn(baseMessage, "Not authorized address tried to increase Locking Cap. Address: ", tx.getSender(signatureCache));
             return false;
         }
 
         // New Locking Cap must be bigger than current Locking Cap
         Optional<Coin> currentLockingCap = getLockingCap();
 
         if (!currentLockingCap.isPresent()) {
-            logger.warn("[increaseLockingCap] {}", "Current Locking Cap is not set since RSKIP134 is not active");
+            logger.warn(baseMessage, "Current Locking Cap is not set since RSKIP134 is not active. Value attempted: ", newLockingCap.value);
             return false;
         }
 
         Coin lockingCap = currentLockingCap.get();
 
         if (newLockingCap.compareTo(lockingCap) < 0) {
-            logger.warn("[increaseLockingCap] {} {}", "Attempted value doesn't increase Locking Cap. Value attempted: ", newLockingCap.value);
+            logger.warn(baseMessage, "Attempted value doesn't increase Locking Cap. Value attempted: ", newLockingCap.value);
             return false;
         }
 
         Coin maxLockingCapVoteValueAllowed = lockingCap.multiply(constants.getIncrementsMultiplier());
         if (newLockingCap.compareTo(maxLockingCapVoteValueAllowed) > 0) {
-            logger.warn("[increaseLockingCap] {} {}", "Attempted value increases Locking Cap above its limit. Value attempted: ", newLockingCap.value);
+            logger.warn(baseMessage, "Attempted value increases Locking Cap above its limit. Value attempted: ", newLockingCap.value);
```

Comment:
```suggestion
            logger.warn(baseMessage, "Attempted value tries to increase Locking Cap above its limit. Value attempted: ", newLockingCap.value);
```

Should we also show the `maxLockingCapVoteValueAllowed`?

Reply:
Done!